src/components/SRNEntry.vue

```diff
@@ -9,24 +9,35 @@
       >
         <input
           v-model="input.userID"
-          type="text"
+          type="tel"
           inputmode="numeric"
           pattern="[0-9]*"
           placeholder="Your SRN / आपका SRN"
           required
           @keypress="isValidSRNFormat($event)"
           class="inputStyleClass"
-          @input="updateValue"
+          @input="updateValue($event, index)"
         />
-       </div>
-       
+        <div
+          class="minus-sign"
+          v-show="ifUserEnteredMoreThanOne"
+          @click="removeField(index, userIDList)"
+        ></div>
+      </div>
+
       <span class="errorStyleClass" v-if="invalidInputMessage">{{
         invalidInputMessage
       }}</span>
       <span class="errorStyleClass" v-if="!isUserValid && validateCount == 1">{{
         invalidLoginMessage
       }}</span>
 
+      <div class="flex flex-row my-auto multiple-div" v-if="isUserValidated">
```

Comment:
This should check if all users before are validated right? Or, if you only let them add another user after the first one, that the last one is validated.

Another thing you should do is to make the valid users uneditable. Once added, is done

Reply:
Maybe this is what is being done already looking at `isUserValidated`? Is this deployed on staging?

Reply:
Yeah, that's is happening, but it's happening in the `addField` method. SRN gets authenticated there.

Reply:
I didn't say that correctly. There is a method `authenticateSRN`. `addField` calls this method as it is a pre-requisite for adding another SRN.